stable/sysdig/templates/daemonset.yaml

```diff
@@ -1,8 +1,43 @@
 {{- if .Values.sysdig.AccessKey }}
+---
+{{- if .Values.kubernetes.rbac }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.sysdig.serviceAccountName }}
+  namespace: {{ .Release.Namespace }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: sysdig-cluster-role
+rules:
+  - apiGroups: ["*"]
+    resources: ["*"]
+    verbs: ["*"]
+  - nonResourceURLs: ["*"]
+    verbs: ["*"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: sysdig-cluster-role-binding
+  namespace: {{ .Release.Namespace }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Values.sysdig.serviceAccountName }}
+    namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: sysdig-cluster-role
+  apiGroup: rbac.authorization.k8s.io
+{{- end }}
+---
 apiVersion: extensions/v1beta1
 kind: DaemonSet
 metadata:
   name: {{ template "fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
When helm installing with a defined namespace, this needs to be in place so that the daemonset gets deployed in that namespace. Otherwise it will not, i have tested this out with secrets, deployments, daemsonsets and without this change things get deployed in the wrong namespaces.